src/db.rs

```diff
@@ -1100,6 +1121,56 @@ impl CommitOverlay {
 	}
 }
 
+#[derive(PartialEq, Eq)]
+pub enum Change<Key, Value> {
+	SetValue(Key, Value),
+	RemoveValue(Key),
```

Comment:
`Dereference`

Reply:
Here I previously got two different operattion: Remove and DecRc, since they are exactly the same I remove one.
But If I want `Dereference` I would probably need to also use `Remove`. (or we go back to a semantic where Set uses Option)

Reply:
One operation is OK, but I think it should be called `Dereference`. For non-ref counted columns you can still think it removes a single reference and removes the value.